service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/RoboticsController.java

```diff
@@ -0,0 +1,40 @@
+package uk.gov.hmcts.reform.fpl.controllers;
+
+import io.swagger.annotations.Api;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.http.HttpStatus;
+import org.springframework.security.access.annotation.Secured;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.server.ResponseStatusException;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.fpl.service.ccd.CoreCaseDataService;
+import uk.gov.hmcts.reform.fpl.service.robotics.RoboticsNotificationService;
+
+import static org.apache.commons.lang3.ObjectUtils.isEmpty;
+
+@Api
+@RestController
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+@ConditionalOnProperty(prefix = "feature.toggle", name = "robotics.support.api.enabled", havingValue = "true")
+public class RoboticsController {
+    private final CoreCaseDataService coreCaseDataService;
+    private final RoboticsNotificationService roboticsNotificationService;
+
+    @PostMapping("/sendRPAEmailByID/{caseId}")
+    @Secured("caseworker-publiclaw-systemupdate")
+    public void resendCaseDataNotification(@PathVariable ("caseId") String caseId) {
+        CaseDetails caseDetails = coreCaseDataService.findCaseDetailsById(caseId);
+
+        if (isEmpty(caseDetails)) {
```

Comment:
Do you think it be more obvious about what is happening if we just do a `== null` here?
One thing I dislike about our use of this isEmpty method is that whilst it can handle objects in general, it's name (and implementation) makes me think it is more suited to dealing with Collections/Arrays/Strings etc.
`isEmpty` to me on a general object makes me think that the method will be going through all properties on the method to check that all of them are null, but maybe that is just me 🤷‍♂